README.md

```diff
@@ -2,11 +2,59 @@
 title: Firebolt OpenRPC Tools
 ---
 # Firebolt OpenRPC Tools
-Tools to enable consistent Firebolt SDK generation.
+Tools to enable consistent Firebolt SDK and document generation.
 
-## Usage
-To install the package, run:
+## To Use
 
-```
-npm install @firebolt-js/openrpc
-```
\ No newline at end of file
+  - First, `npm install`.
+  - Next, invoke the cli directly with `src/cli.js`. See below for arguments.
+
+Alternatively, you may install globally using `npm install -g @firebolt-js/openrpc`, which will install the cli and add it to your $PATH so it may be invoked with the `firebolt-openrpc` command. However, trying this at the time of writing, it did not work.
+
+## CLI Arguments
+
+`--task`: Tell the tool what you want it to do. Options are:
+
+  - `sdk`: Generate the SDK
+  - `docs`: Generate the docs
+  - `openrpc`: Generate the openrpc
+  - `validate` Validate the openrpc
+
+`--source`: The relative or absolute path to the folder or file that the task will use.
+
+`--template`: The relative or absolute path to the folder or file containing template resources for the task.
+
+`--output`: The relative or absolute path to the folder or file that will hold the task's generated output.
+
+`--as-path`: This has some effect on the document generation task. Not sure what.
```

Comment:
This is a toggle for generating content as files vs folders, e.g.:

`/docs/Topic.md`

vs

`/docs/Topic/index.md`

It effects output of:

- file names
- relative links
- relative images

Reply:
When deploying docs to web servers, --as-path is generally used. For deploying to GitHub pages/wikis, do not use --as-path.

Reply:
Perfect. Thanks for the clarification. I will update.